upstream-tools/package.json

```diff
@@ -8,7 +8,7 @@
     "strip-links": "tf-patch strip-links --cwd ../upstream",
     "global-replace": "tf-patch global-replace --cwd ../upstream",
     "replace": "tf-patch replace --cwd ../upstream",
-    "apply": "yarn pre-replace && yarn strip-links && yarn global-replace"
```

Comment:
nice! I wonder if we should reformat our patches as well.

Right now when working on a patch we'd most likely get the upstream tests complaining about formatting, right?

Reply:
I've started going that route, but with this addition it's no longer necessary to do that, just busy-work, so I think we do not need to reformat the patches.

Reply:
But if I'm currently rebasing the patches or working on one, this tool wouldn't be executed. Right? Afaik it's only executed on `make upstream`

In this case we'd most likely have formatting issues?:
```
./upstream.sh check_out
./upstream.sh rebase -i
```